Package.swift

```diff
@@ -66,11 +69,17 @@ let package = Package(
             dependencies: [
                 "SwiftDocCUtilities",
                 "SwiftDocC",
+                "SwiftDocCTestUtilities",
             ],
             resources: [
                 .copy("Test Resources"),
                 .copy("Test Bundles"),
             ]),
+        
+        // Test utility library
+        .target(
+            name: "SwiftDocCTestUtilities",
```

Comment:
Right now this target is only used to share one file between the two test targets but there are more code that could be moved out of SwiftDocC itself into this test target instead.

Reply:
This is great! I've been wanting something like this for a while. No need to do it now- but eventually moving all of our test bundles over to this library would have a big impact on our repo size I think.